src/common/config/env.validation.ts

```diff
@@ -98,78 +106,102 @@ export class EnvironmentVariables {
   @Transform(({ value }) => value || LogFormat.json)
   LOG_FORMAT: LogFormat = LogFormat.json;
 
+  @IsNotEmpty()
   @IsArray()
   @ArrayMinSize(1)
-  @Transform(({ value }) => value.split(',').map((url) => url.replace(/\/$/, '')))
+  @IsUrl({
+    require_protocol: true
+  }, {
+    each: true,
+  })
+  @Transform(({ value }) => toArrayOfUrls(value))
   PROVIDERS_URLS!: NonEmptyArray<string>;
 
-  @IsInt()
+  @IsNotEmpty()
+  @IsEnum(Network)
   @Transform(({ value }) => parseInt(value, 10))
-  CHAIN_ID!: number;
+  CHAIN_ID!: Network;
 
+  @IsNotEmpty()
```

Comment:
Did you check how it behave now without @IsNotEmpty() check ?

Reply:
If we don't set `IsNotEmpty()` here it will work as follows.

1. If the `DB_HOST` variable is not presented in the `.env` file, the validation will throw an error, because `undefined` is not a string.

2. If a user sets an empty string in the `.env` file like this: `DB_HOST=`, the validation error will not be thrown, because the empty string is a string, everything is OK.

I don't think that we want to allow empty strings here.